packages/components/src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useTabList } from '@react-aria/tabs';
+import {
+  Box,
+  ThemeExtensionsWithParts,
+  useComponentStyles,
+} from '@marigold/system';
+import { useTabListState } from '@react-stately/tabs';
+import { useRef } from 'react';
+import { AriaTabListProps } from '@react-types/tabs';
+
+import { Tab } from './Tab';
+import { TabPanel } from './TabPanel';
+import { TabContext } from './Context';
+import { Item } from '@react-stately/collections';
+
+export interface TabsThemeExtension
+  extends ThemeExtensionsWithParts<'Tabs', ['tabs', 'tab', 'tabPanel']> {}
+export interface TabsProps
+  extends Omit<AriaTabListProps<object>, 'orientation'> {
```

Comment:
`react-aria` uses a lot of `isDisabled` props (basically they all start with "is"), we use the HTML names. So instead of `isDisabled` the component should accept `disabled`. And ofc it then needs to be cast inside the component :D

Reply:
yes this will be better and more convenient, 
now it accepts disabled